Automattic/newspack-blocks
# Deduplication - use same store form Carousel and Homepage Posts blocks; preview post in block previews

### All Submissions:






### Changes proposed in this Pull Request:



1. Closes #664 
2. Makes the block previews use a static post, instead of fetching actual posts

### How to test the changes in this Pull Request:

1. On `master`, create a page with both Carousel and Homepage Posts blocks interspersed
2. Observe that posts are duplicated (in the editor and on the front-end)
3. Switch to this brach, rebuild
4. Observe there is no duplication now ✨
5. After selecting a Homepage posts block in the editor, observe the block preview in the sidebar - it should display a hardcoded preview post
6. Trigger post pattern insertion, observe the pattern previews use a hardcoded preview post

### Other information:
